homeassistant/components/wiz_light/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""WiZ Light integration."""
+import logging
+
+from pywizlight import wizlight
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["light"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Old way of setting up the wiz_light component."""
+    hass.data[DOMAIN] = {}
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up the wiz_light integration from a config entry."""
+    ip_address = entry.data.get(CONF_HOST)
+    _LOGGER.debug("Get bulb with IP: %s", ip_address)
+    bulb = wizlight(ip_address)
```

Comment:
You should probably surround this call to wizlight with a try-except block as well just as you do in the config flow. The device may have gone offline since the config entry was created or may be unreachable for some other reason. Checking it here and returning false notifies home assistant that the entryfailed to setup.